packages/core/src/main/client/ClientDatabaseRepository.ts

```diff
@@ -55,7 +55,7 @@ class ClientDatabaseRepository {
   }
 
   public createClientList(userId: string, clientList: RegisteredClient[]): Promise<MetaClient[]> {
-    const createClientTasks: Promise<MetaClient>[] = [];
+    const createClientTasks: Array<Promise<MetaClient>> = [];
```

Comment:
Why are you doing the reverse here then?

Reply:
Because it's not a simple type.

Reply:
Is `Array<Promise<MetaClient>>` different from `Promise<MetaClient>[]`?

Reply:
No, it's just a a different style of writing.

Reply:
Ok, then let's use `[]` for everything (no matter if complex type or simple type).

Reply:
Okay!

Reply:
Will you update the PR then?

Reply:
yes :)